atomics/T1528/T1528.yaml

```diff
@@ -0,0 +1,57 @@
+attack_technique: T1528
+display_name: Steal Application Access Token
+atomic_tests:
+- name: Azure - Dump All Azure Key Vaults with Microburst
+  description: Upon successful execution of this test, the names, locations, and contents of key vaults within an Azure account will be output to a file.
+  See: https://www.netspi.com/blog/technical/cloud-penetration-testing/a-beginners-guide-to-gathering-azure-passwords/
```

Comment:
@Leomon5 : can you edit this line `See:` , the : is treated as a new key name in this yaml. Use a hyphen instead

Reply:
Hey @patel-bhavin , this should be fixed now. Thank you for pointing it out!